homeassistant/components/edl21/sensor.py

```diff
@@ -269,9 +260,25 @@ async def async_setup_platform(
     config: ConfigType,
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up EDL21 sensors via configuration.yaml."""
+
+    hass.async_create_task(
```

Comment:
I think the old schema needs to be kept, and it should raise a deprecation warning before import.

Reply:
Ok, I added a deprecation warning.